app/helpers/fae/form_helper.rb

```diff
@@ -139,6 +141,13 @@ def label_and_hint(attribute, options)
       options[:hint] = hint.html_safe if hint.present?
     end
 
+    def translate_button(f, attribute, options)
+      if Fae::Option.instance.translate_language && !attribute.to_s.include?('_en') && Fae.languages.keys.any? { |lang| attribute.to_s.include?(lang.to_s) }
+        translate = content_tag(:span, 'Translate', class: 'button js-translate-button', style: 'position: absolute; margin-left: 10px;')
```

Comment:
Plz make this styling a CSS class instead of inlining it here. There's a couple more instances of this happening that can be addressed as well

Reply:
updated!